spec/requests/graphql/mutations/create_user_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe 'create user' do
+  it 'can create a user' do
+    post '/graphql', params: { query: query }
+
+    expect(User.count).to eq(1)
+  end
+
+  def query
+      <<~GQL
+      mutation{
```

Comment:
At some point I'd like to look into why we have to use input & params here. May make it easier to use the mutations. I'm not sure what causes us to need it...

Reply:
agreed. I know where params come from though, well 90% sure. The argument line in the mutation files. Params are what we currently use, but we can change it to whatever. 

 Maybe the input come from us using input types? idk.